plugins/modules/net_tools/nmcli.py

```diff
@@ -95,6 +95,47 @@
         type: list
         elements: str
         version_added: 2.0.0
+    routes4_extended:
+        description:
+            - The list of ipv4 routes.
```

Comment:
```suggestion
            - The list of IPv4 routes.
            - Mutually exclusive with I(routes4).
```
Also mention that this one exists in the `routes4` documentation.